### PR TITLE
chore: make examples take only as much space as necessary

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/addon-doc/components/example/example.style.less [hunk 1]
```diff
@@ -56,6 +56,8 @@
 
 .t-demo {
     .customize-scroll();
+    display: flex;
+    align-items: flex-start;
```
Comment (nsbarsukov) [inline projects/addon-doc/components/example/example.style.less:60]: Many example has problems with layout now :(

For example: https://taiga-ui--pr4905-e2e-size-p44bcl8e.web.app/components/input-number#sizes
Initially: https://taiga-ui.dev/components/input-number#sizes

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/addon-doc/components/example/example.style.less (hunk 1 above)
Comment (waterplea) [reply projects/addon-doc/components/example/example.style.less:60]: Oh, I forgot `flex-direction`. Still broken though, I'll think about it.

**Comment by splincode** [comment]
image


https://github.com/Tinkoff/taiga-ui/actions/runs/5555014586?pr=4905

image

**Review by vladimirpotekhin — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Comment by nsbarsukov** [comment]
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/button#padding
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/input-inline#base
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/slider#key-steps
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/input#autofill
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/progress-segmented
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/components/progress-bar 
* https://taiga-ui--pr4905-e2e-size-3tsjvzgk.web.app/charts/line-days-chart#base

**Pull request merged**